Domains/2-DisciplinePhysical/StructuralPhysical/StructuralPhysical.remarks.md

```diff
@@ -7,8 +7,90 @@ remarksTarget: StructuralPhysical.ecschema.md
 
 This schema contains classes that are used to model the real-world physical entities that comprise the structural systems of infrastructure.
 
-## StructuralPhysicalModel
+## Entity Classes
+
+### StructuralPhysicalModel
 
 ***This class has been deprecated. Do not use.***
 
-Applications, services and components should never create instances of this class. Use `PhysicalModel` instead.
\ No newline at end of file
+Applications, services and components should never create instances of this class. Use `PhysicalModel` instead.
+
+### StructuralElement
+The abstract class `StructuralElement` is the generalization of structural member.
```

Comment:
Since by "structural member", the text refers to a class, it should be named accordingly and formatted like `StructuralMember`.

Reply:
Fixed